web/source/osk/oskKey.ts

```diff
@@ -113,7 +113,23 @@ namespace com.keyman.osk {
       '*ZWNJiOS*':        0x75, // The iOS version will be used by default, but the
       '*ZWNJAndroid*':    0x76, // Android platform has its own default glyph.
     };
-    
+
+    static readonly BUTTON_CLASSES = [
+      'default',
+      'shift',
+      'shift-on',
+      'special',
+      'special-on',
+      '',
+      '',
+      '',
```

Comment:
Can we document why these three are blank?

Reply:
That's a good question.  I assume they're "reserved"?  They've been in there since KMW 2.0 at a minimum, so this predates my time on the project, so I've just been treating them as "reserved."